packages/core/src/common/utils/jsUtils.ts

```diff
@@ -79,3 +79,10 @@ export function uniqueId(namespace: string) {
     uniqueCountForNamespace.set(namespace, curCount + 1);
     return `${namespace}-${curCount}`;
 }
+
+/**
+ * @returns `true` if the value is an empty string after trimming whitespace
+ */
+export function isEmptyString(val: any) {
```

Comment:
this should either get used in `reactUtils.ts` (which was the original intent) or be removed